packages/@aws-cdk-testing/framework-integ/test/lambda-layer-awscli/test/integ.awscli-layer.js.snapshot/lambda-layer-awscli-integ-stack.template.json

```diff
@@ -12,7 +12,7 @@
     "Description": "/opt/awscli/aws"
    }
   },
-  "Lambdapython37ServiceRoleB5A704D4": {
+  "Lambdapython39ServiceRoleE2CFED77": {
```

Comment:
Expected change, since we updated python runtime to 3.9 and 3.10 for the test. 3.7 is deprecated and results in a deployment failure.